vllm/platforms/cuda.py

```diff
@@ -139,7 +139,10 @@ def check_and_update_config(cls, vllm_config: "VllmConfig") -> None:
 
         # TODO(lucas): handle this more gracefully
         # Note: model_config may be None during testing
-        if model_config is not None and model_config.use_mla:
+        # Note(Zhiyuan): for MLA, actual logic block size will be determined at
+        # vllm/v1/worker/gpu_model_runner.py
```

Comment:
I think we don't need to mention logic block size in this comment. Just mention block size is initialized in `HybridAttentionMambaModelConfig.verify_and_update_config` and doesn't need to reinitialize here.